invenio_deposit/bundles.py

```diff
@@ -37,12 +41,27 @@
         "ckeditor": "latest",
         "flight": "latest",
         "eonasdan-bootstrap-datetimepicker": "4.14.30",
+        "base-64": "latest",
+        "json-editor": "latest",
```

Comment:
Yes. This is for manual&&deposit. Nobody will type-in 3000 authors (or other huge documents) by hand. *knight* is only for editor records that already exists and for power users. json-editor is really good for the deposit case. I'll prepare a new module for the power-user editor (aka *knight*).